doc/release-notes.md

```diff
@@ -28,22 +35,378 @@ reindex or re-sync the whole chain.
+---------------
+- When creating a hex-encoded Dash transaction using the `dash-tx` utility
+  with the `-json` option set, the following fields: `addresses`, `reqSigs` are no longer
+  returned in the tx output of the response.
+  
+dash-cli Changes
+----------------
+
+- A new `-rpcwaittimeout` argument to `dash-cli` sets the timeout
+  in seconds to use with `-rpcwait`. If the timeout expires,
+  `dash-cli` will report a failure. (dash#5923)
+- The `connections` field of `dash-cli -getinfo` is expanded to return a JSON
+  object with `in`, `out` and `total` numbers of peer connections and `mn_in`,
+  `mn_out` and `mn_total` numbers of verified mn connections. It previously
+  returned a single integer value for the total number of peer connections. (dash#5823)
+- Update `-getinfo` to return data in a user-friendly format that also reduces vertical space.
+- CLI `-addrinfo` now returns a single field for the number of `onion` addresses
+  known to the node instead of separate `torv2` and `torv3` fields, as support
+  for Tor V2 addresses was removed from Dash Core in 18.0.
+
+dash-wallet changes
+-------------------
+This release introduces several improvements and new features to the `dash-wallet` tool, making it more versatile and
+user-friendly for managing Dash wallets.
+
+Wallets created with the `dash-wallet` tool will now utilize the `FEATURE_LATEST` version of wallet which is the HD
+(Hierarchical Deterministic) wallets with HD chain inside.
+
+- new command line argument `-usehd` which let to create non-Hierarchical Deterministic (non-HD) wallets with the
+  `wallettool` for compatibility reasons since default version is bumped to HD version
+- new command line argument `-descriptors` to enable _experimental_ support of Descriptor wallets. It lets users to
+  create descriptor wallets directly from the command line. This change aims to align the command-line interface with
+  the `createwallet` RPC, promoting the use of descriptor wallets which offer a more robust and flexible way to manage
+  wallet addresses and keys.
+- new commands `dump` and `createfromdump` have been added, enhancing the wallet's storage migration capabilities. The
+  `dump` command allows for exporting every key-value pair from the wallet as comma-separated hex values, facilitating a
+  storage agnostic dump. Meanwhile, the `createfromdump` command enables the creation of a new wallet file using the
+  records specified in a dump file. These commands are similar to BDB's `db_dump` and `db_load` tools and are crucial 
+  for manual wallet file construction for testing or migration purposes.
+
+
+P2P and network changes
+-----------------------
 
-A deadlock caused nodes to become non-responsive and RPC to report "Work depth queue exceeded".
-Thanks to Konstantin Akimov (knst) who discovered the cause. This previously caused masternodes to become PoSe banned.
+- The protocol version has been bumped to 70232. This should be identical in behavior to 70231.
```

Comment:
Maybe clarify why it was bumped if there's no change in behavior
```suggestion
- The protocol version has been bumped to 70232. This should be identical in behavior to 70231.
```

Reply:
We bump it to distinguish between v20 / v21. Not sure best way to communicate that or if it's important.

Reply:
Made a suggestion